submodules/linphone

```diff
@@ -1 +1 @@
-Subproject commit 82e395bc9b140f7f97b7ab9b41d8e5aa41f70965
+Subproject commit 0e996357d70a606a86d97af7d28eaa8451f09952
```

Comment:
The diff between these two shows some pretty substantial vCard changes.

I've built it locally, and it does build and run, but we really need to stop following upstream so closely if we hope to release GA soon.

This means: no more upstream merges for now, unless we run into some huge problems that require it. Ok?